app/dashboard/templates/profiles/profile.html

```diff
@@ -47,7 +47,7 @@
       <div class="container profile-card mt-md-n5 mh-150">
         <div class="row">
           {% if total_kudos_count %}
-            <div id=kudos_header>
+            <div id="kudos_header" class="d-md-block d-none">
```

Comment:
per https://getbootstrap.com/docs/4.3/utilities/display/  shouldn t this be `.d-sm-none .d-md-block` to just hide on mobile?

Reply:
better yet why dont we just hide 2 of the kudos so it displays/aligns on mobile?

Reply:
actually no, because bootstrap is mobile first. this will showed on min like this:
![image](https://user-images.githubusercontent.com/1358093/66070920-956e5800-e528-11e9-9f36-c19e6c323a01.png)
Smaller than that will be hidden 
![image](https://user-images.githubusercontent.com/1358093/66070965-a919be80-e528-11e9-99f1-1bfd36437e4d.png)

Reply:
@owocki or maybe just :
![image](https://user-images.githubusercontent.com/1358093/66071303-5096f100-e529-11e9-9eae-302e427c18a6.png)

Reply:
Just to make sure I am on the same page here, are you now wanting a redesign of how kudos badges are displayed rather than just hiding them?

Reply:
Let attach to the original ticket, unless @owocki have any suggestion. 
Just cleanup the changes not related to the ticket on the commit and I believe we are good.